src/pages/Validator/Staking.tsx

```diff
@@ -0,0 +1,51 @@
+import {
```

Comment:
On a second thought, since not all validator has its own page, this folder is more about Delegator. So maybe `src/pages/Delegator` is a better name for the folder. We don't need to change it in this PR though. Just throwing the thought here.

Reply:
good callout, we can rename it to something like `delegatoryvalidator`? cuz `delegator` is a diff concept from `validator`, they delegate money to the validator pool, but this page is more for the delegatory validator who can be delegated.